tools.py

```diff
@@ -501,13 +528,28 @@ async def send_paginated_embed(bot:  discord.ext.commands.Bot,
         page_text = PAGE_TEMPLATE.format(current_page, len(pages))
         embed.title = f'{title} {page_text}'
 
-        if len(pages) == 1: # Short circuit: if one page only, we don't need to change pages
+        if single_page or dm_channel:
             embed.set_footer(text=page_text)
             await channel.send(embed=embed)
+
+        if single_page:
             break
+
+        elif dm_channel:
+            if current_page >= 10:
+                if len(pages) > 10:
+                    await channel.send(f'Limited to 10 pages in DM channel. {len(pages) - 10} page{"s" if len(pages) == 1 else ""} was not sent')
```

Comment:
Minor gripe, "was not sent" should be updated to use was/were

Reply:
Resolved in 79046a7